spinnaker/spinnaker_system/google_http_lb_upsert_test.py

```diff
@@ -134,31 +140,34 @@ def scenario(self):
   def test_c_upsert_min_lb(self):
     self.run_test_case(self.scenario.upsert_min_load_balancer())
 
-  def test_p_upsert_full_lb(self):
+  def test_d_upsert_full_lb(self):
     self.run_test_case(self.scenario.upsert_full_load_balancer())
 
-  def test_s_change_hc(self):
+  def test_e_add_security_group(self):
+    self.run_test_case(self.scenario.add_security_group())
+
```

Comment:
Mention 'f' is reserved for the other fixture to inject its test into this part of the sequence.

The naming here is going to be brittle, especially the way we're interleaving classes and given the number of tests.

It looks like there are 3 phases of tests (which is typical)
creation
usage
deletion

And furthermore, the usage is  partitioned into modifying the different aspects.
I wonder if you added another grouping decorator then that would reduce the brittleness to firewall changes within a group.

For example, adding another thing in the creation phase wouldnt cascade through the rest of the tests. If it could be added at the end, it would have no impact at all.

Assuming ordering doesnt matter (or you could decorate again after the phase name)
def test_phaseA_creation_upsert_min_...
def test_phaseB_creation_upsert_full_...

The modifications are going to be order sensitive
def test_phaseC_modify_a_...

What do you think?

You can leave a whole in a phase, or a whole for a completely new phase, for use by the subclass depending on how you view it.